Gemfile.lock

```diff
@@ -24,6 +24,7 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
I suspect a11ed7c1b942f2a262ce74cf402637503d15a13d should be reverted. I'm not sure why a translation would include node module updates. Am I missing something?

Reply:
It was a mistake. Sorry.

I think now it's ok.

Thank you.